WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Exceptions/DevicePortalException.cs

```diff
@@ -31,6 +31,16 @@ namespace Microsoft.Tools.WindowsDevicePortal
 #endif // !WINDOWS_UWP
     public class DevicePortalException : Exception
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DevicePortalException"/> class.
+        /// </summary>
+        /// <param name="baseException">A base exception that we are converting to a DevicePortalException.</param>
+        public DevicePortalException(Exception baseException)
```

Comment:
We should retain the ability for applications to specify their own messages. This is used extensively in the Mixed Reality Commander app (https://github.com/Microsoft/MixedRealityCompanionKit/tree/master/HoloLensCommander) to allow the UI to display information regarding where in the flow an exception occurred.